src/schema/objects/common_principles.yaml

```diff
@@ -44,6 +43,11 @@ data_type:
 
         14. `mrs` (magnetic resonance spectroscopy)
 
+        15. `phenotype` (participant-level measurement and survey data)
+
+    For files that are subject- and session- specific, the data type directory
+    is nested inside those directories.
```

Comment:
Does this mean `phenotype/` is permitted, but not required under the `subject/[session/]` directory? BEP036 is leaning into the aggregated model of tabular phenotypic data file storage at the root-level `phenotype/` subdirectory.

Reply:
To permit it, we would need a file rule where `subject` and `session` are optional, so this describes the current situation reasonably well, as it can't be subject- or session- specific.

I do not have a specific objection to permitting it to be used inside subject/session directories, but if BEP036 is not proposing it, it won't happen.

Happy to consider alternative wordings that make this clearer.